structs.go

```diff
@@ -1446,6 +1446,14 @@ type AutoModerationTriggerMetadata struct {
 	// Internally pre-defined wordsets which will be searched for in content.
 	// NOTE: should be only used with keyword preset trigger type.
 	Presets []AutoModerationKeywordPreset `json:"presets,omitempty"`
+
+	// Substrings which should not trigger the rule.
+	// NOTE: should be only used with keyword or keyword preset trigger type.
+	AllowList []string `json:"allow_list,omitempty"`
```

Comment:
Since allow list can be an empty list, it should be a pointer.

```suggestion
	AllowList *[]string `json:"allow_list,omitempty"`
```

Reply:
OK, fixed